### PR TITLE
Changes the application to use the an S3 bucket to host static files. 

### PR DESCRIPTION
**Pull request by khandelwal**
This will use an S3 bucket in production, but not in development.

### CODE REVIEW

File: foia_hub/templates/includes/footer.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
 <footer>
   <div class="container">
       <a href="https://18f.gsa.gov">
-        <img src="/static/images/logo-18f.png" />
+        <img src="{{ static("images/logo-18f.png") }}" />
```
Comment (khandelwal) [inline foia_hub/templates/includes/footer.html]: We stop hardcoding the static asset URLs here, and start using this template convenience.

**Comment by khandelwal** [comment]
Fixes: #489

**Comment by cmc333333** [comment]
We needed to tweak a few additional bits (like https://github.com/18F/peacecorps-site/blob/master/peacecorps/peacecorps/settings/production-admin.py#L22). Just a heads up if you haven't tested this yet.

**Comment by khandelwal** [comment]
Good catch, @cmc333333 I forgot that one. I have tested it with production settings, and with development settings and didn't see anything broken. But I've seen in multiple places that having that parameter is useful.

File: foia_hub/settings/production.py [hunk 2]
```diff
@@ -11,6 +11,14 @@
 # if testing out production settings in development:
 # ALLOWED_HOSTS = ['foia.18f.us', 'localhost']
 
+AWS_STORAGE_BUCKET_NAME = 'openfoia-static'
```
Comment (ramirezg) [inline foia_hub/settings/production.py]: Should we keep the bucket name in an environment env as well?

**Comment by ramirezg** [comment]
@khandelwal I just hooked this PR into my personal aws account. Everything looks fine expect for the picture icons. Are these stored in a place were `collectstatic` can't reach them?
screen shot 2015-03-05 at 12 10 29 pm

**Comment by khandelwal** [comment]
This has to do with the CORs configuration on your bucket. Do you see *any* of the font-awesome icons?

**Comment by ramirezg** [comment]
@khandelwal none, how should they be set?

**Pull request merged**